pkg/create/sample-cluster.go

```diff
@@ -0,0 +1,154 @@
+/*
+Copyright 2023 The Rook Authors. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package create
+
+import (
+	"bytes"
+	"context"
+	_ "embed"
+	"fmt"
+	"io"
+	"net/http"
+	"os"
+	"strings"
+	"text/template"
+
+	"github.com/rook/kubectl-rook-ceph/pkg/exec"
+	"github.com/rook/kubectl-rook-ceph/pkg/k8sutil"
+	"github.com/rook/kubectl-rook-ceph/pkg/logging"
+)
+
+type config struct {
+	ClusterNamespace string
+	StorageClassName string
+}
+
+var (
+	githubURL = "https://raw.githubusercontent.com/rook/rook/master/deploy/examples/"
+	//go:embed test-cluster-on-pvc.yaml
+	clusterYaml string
+)
+
+func SampleCluster(ctx context.Context, clientsets *k8sutil.Clientsets, operatorNamespace, clusterNamespace, storageClassName string) {
+	var answer, output string
+	logging.Warning(`
+	"This command will create a single-node Rook-Ceph cluster. We assume that you don't already have a Rook-Ceph cluster.
```

Comment:
How about also warning them that backing storage is needed from a storage class?